frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -353,6 +373,13 @@ export const CurrentVersion: React.FC<CurrentVersionProps> = ({ cv }) => {
 };
 
 export const UpdateLink: React.FC<CurrentVersionProps> = ({ cv, clusterVersionIsEditable }) => {
+  // assume if 'worker' is editable, others are too
+  const workerMachineConfigPoolIsEditable = useAccessReview({
+    group: MachineConfigPoolModel.apiGroup,
+    resource: MachineConfigPoolModel.plural,
+    verb: 'patch',
+    name: 'worker',
+  });
```

Comment:
Is this an assumption we should be making? Can't RBAC checks deny access partially to some resources? Are we assuming this will always be a super admin so if they have one access they'll have all?

Reply:
Yes to the assumption the user is a super admin.  We don't know about the others because they are optional.